mod/InGameTracker/TrackerInventoryMode.cs

```diff
@@ -0,0 +1,134 @@
+        {
+            Randomizer.OWMLModConsole.WriteLine("Tracker Mode created");
+        }
+
+        // Runs when the mode is opened in the ship computer
+        public override void EnterMode(string entryID = "", List<ShipLogFact> revealQueue = null)
+        {
+            Tracker.CheckInventory();
+            if (Wrapper == null)
+            {
+                Randomizer.OWMLModConsole.WriteLine("Wrapper is null!", OWML.Common.MessageType.Error);
+            }
+            else
+            {
+                Randomizer.OWMLModConsole.WriteLine("Opened Inventory Mode", OWML.Common.MessageType.Info);
+            }
+            Wrapper.Open();
+            Wrapper.SetName("AP Inventory");
+            Wrapper.SetItems(Tracker.InventoryItems);
+            Wrapper.SetSelectedIndex(0);
+            Wrapper.UpdateList();
+            selectedIndex = 0;
+            RootObject.name = "ArchipelagoTrackerMode";
+
+            SelectItem(0);
+        }
+
+        // Runs when the mode is closed
+        public override void ExitMode()
+        {
+            string[] keys = Tracker.NewItems.Keys.ToArray();
+            foreach (string key in keys) Tracker.NewItems[key] = false;
+            Wrapper.Close();
+        }
+
+
+        // Runs when player enters computer, update info that changes between computer sessions. Runs after EnterMode
+        public override void OnEnterComputer()
+        {
+
+        }
+
+        // Runs when the player exits the computer, after ExitMode
+        public override void OnExitComputer()
+        {
+
+        }
+
+        // Runs every frame the mode is active
+        public override void UpdateMode()
+        {
+            int changeIndex = Wrapper.UpdateList();
+
+            if (changeIndex != 0)
+            {
+                selectedIndex += changeIndex;
+
+                if (selectedIndex < 0) selectedIndex = Tracker.InventoryItems.Count - 1;
+                if (selectedIndex >= Tracker.InventoryItems.Count) selectedIndex = 0;
+
+                SelectItem(selectedIndex);
+            }
+        }
+
+        // Allows leaving the computer in this mode
+        public override bool AllowCancelInput()
+        {
+            return true;
+        }
+        
+        // Allows swapping modes while in this mode
+        public override bool AllowModeSwap()
+        {
+            return true;
+        }
+
+        // Returns the ID of the selected ship entry, used for knowing which entry should be highlighted when switching to Map Mode. Useless for us probably.
+        public override string GetFocusedEntryID()
+        {
+            return "";
+        }
+
+        // Shows the item selected and the associated info
+        private void SelectItem(int index)
+        {
+            string itemID = Tracker.ItemEntries.ElementAt(index).Key;
+            Sprite tex = TrackerManager.GetSprite(itemID);
```

Comment:
Wouldn't `tex` be for a texture, not a sprite?

Reply:
Yes but originally this returned a Texture2D and then I realized I needed to change the types to a sprite for them to work, and I just didn't bother renaming the variables I had.